### DISCUSSION TITLE
How to modify the margins or the width of the side panels?

### DISCUSSION THREAD

**Discussion by Masber** [squidfunk/mkdocs-material#7272] [Q&A] [answered]
Dear material mkdocs community,

I am learning material mkdocs and have this example https://eth-cscs.github.io/manta/basics/

I have some code in there which won't fit in the site's content section, my impressions is this could be fixes by reducing the width of the side panels `navigation` and `table of content`.

Is there a way to reduce the width of the side panels?

thank you very much

**Comment by squidfunk** [comment, marked as answer]
You can [increase the size of the content area width](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#content-area-width) with [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css):

``` css
.md-grid {
  max-width: 1440px; 
}
```

**Reply by kamilkrzyskow** [reply]
This was asked before, you can also set the width to `initial` to allow for a full stretch.
- https://github.com/squidfunk/mkdocs-material/discussions/6404#discussioncomment-7695243
